include/world_builder/features/subducting_plate_models/temperature/mass_conserving.h

```diff
@@ -110,7 +110,7 @@ namespace WorldBuilder
             bool adiabatic_heating;
             std::vector<std::vector<Point<2>>> mid_oceanic_ridges;
             Operations operation;
-
+            std::string reference_model_name;
```

Comment:
Please make this an enum so that we don't have to do a lot of string comparisons in while running.